src/main/java/emissary/server/EmissaryServer.java

```diff
@@ -366,6 +366,12 @@ public static void stopServer(final String name, final boolean force, final bool
         }
         LOG.info("Done stopping all places");
 
+        try {
+            MetricsManager.lookup().shutdown();
+        } catch (Exception ex) {
+            LOG.warn("No metrics manager available");
+        }
+
```

Comment:
I think the RollManager.shutdown(); from 384 needs to move ahead of this block as well. 

Stop places, roll everything, then stop metrics, then stop resource watcher.

Reply:
oops agreed, I'll make that change quick